lib/conjur/api/variables.rb

```diff
@@ -105,6 +105,10 @@ def variable_values(varlist)
       end
     end
 
+    def variable_expirations(window = nil)
```

Comment:
Oops, forgot about `window`. It's supposed to get added to the request, I'll fix it.